### ISSUE TITLE
Installation Documentation Adding Symbolic Link for Nginx

### ISSUE THREAD

**Issue by jnewmaster** [digitalocean/netbox#2111]
### Issue type
[ ] Feature request 
[ ] Bug report      
[X] Documentation   

### Environment
* Python version:  3.5.1-3 
* NetBox version:  2.3.3 

### Description

The current documentation for installation > web server > NGINX specifies to create a symbolic link but does not specify the link to create.
Currently states : `ln -s /etc/nginx/sites-available/netbox`
Proposed to state : `ln -s /etc/nginx/sites-available/netbox default`

**Comment by bdlamprecht** [comment]
I think it makes sense the way it is now.

When creating a symbolic link, if you don't specify a different destination filename, it defaults to the source filename.

If you following **exact steps** in the documentation, you should create the file `netbox` in the directory `/etc/nginix/sites-enabled/` which is the idea.

From the documentation under the heading `Web Server Installation`:
```
# cd /etc/nginx/sites-enabled/
# rm default
# ln -s /etc/nginx/sites-available/netbox
```

This will remove the placeholder `default` site with the `netbox` site.
You can always leave the `default` there if you want to or even add additional sites if you need `nginx` to proxy for those as well.

**Comment by jeremystretch** [comment]
Right, we keep the `netbox` name because it's plausible that the user might be serving other sites from this instance as well.

**Comment by jnewmaster** [comment]
Hey Jeremy,

Sure, then wouldn’t it make sense to specify that then?

For example,

“ln -s /etc/nginx/sites-available/netbox netbox”

So that there would be less room for misenterpritstion.

For example, I am not familiar with NGINX but since it is the first listed I chose that as it appeared to be the most supported, but I did not see it specify the link so I was unsure if it should specify “default” or “netbox”.
It wasn’t even until I opened this request that I realized that “netbox” would work with NGINX without further configuration.

While I understand a reader having a better understanding of NGINX here would help, I also believe the added specificity would help.

Thanks for taking the time.
